include/imu_ros2/accelgyrotemp_data_provider.h

```diff
@@ -33,6 +33,7 @@ class AccelGyroTempDataProvider : public AccelGyroTempDataProviderInterface {
 
     void init() override;
     bool getData(imu_ros2::msg::AccelGyroTempData& message) override;
+    bool activateData() override;
```

Comment:
I would rename this to something more suggestive, something like: enableBufferedDataOutput, of course this applies to all instances of activateData